src/libs/PolicyUtils.ts

```diff
@@ -266,9 +266,10 @@ function isPaidGroupPolicy(policy: OnyxEntry<Policy> | EmptyObject): boolean {
 function isTaxTrackingEnabled(isPolicyExpenseChat: boolean, policy: OnyxEntry<Policy>, isDistanceRequest: boolean): boolean {
     const distanceUnit = Object.values(policy?.customUnits ?? {}).find((unit) => unit.name === CONST.CUSTOM_UNITS.NAME_DISTANCE);
     const customUnitID = distanceUnit?.customUnitID ?? 0;
-    const isTaxEnabled = policy?.customUnits?.[customUnitID]?.attributes?.taxEnabled;
     const isPolicyTaxTrackingEnabled = isPolicyExpenseChat && policy?.tax?.trackingEnabled;
-    return (isDistanceRequest ? isPolicyTaxTrackingEnabled && isTaxEnabled : isPolicyTaxTrackingEnabled) ?? false;
+    const isTaxEnabledForDistance = isPolicyTaxTrackingEnabled && policy?.customUnits?.[customUnitID]?.attributes?.taxEnabled;
+
+    return Boolean(isDistanceRequest ? isTaxEnabledForDistance : isPolicyTaxTrackingEnabled);
```

Comment:
Thanks for update! Do we need Boolean()?

Reply:
Yeah - it will break return type - because both isTaxEnabledForDistance and isPolicyTaxTrackingEnabled can be undefined

Reply:
Done!